include/QuartzCore/CAShapeLayer.h

```diff
@@ -34,8 +34,8 @@ CA_EXPORT_CLASS
 
 @property CGPathRef path;
 @property CGColorRef fillColor;
-@property (copy) NSString* fillRule STUB_PROPERTY;
-@property (copy) NSString* lineCap STUB_PROPERTY;
+@property (copy) NSString* fillRule;
```

Comment:
To clarify question above - lets add the variation to woccatalog test to demonstrate this works.